config/swagger-ui.php

```diff
@@ -42,4 +42,26 @@
         'client_id' => env('SWAGGER_UI_OAUTH_CLIENT_ID'),
         'client_secret' => env('SWAGGER_UI_OAUTH_CLIENT_SECRET'),
     ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Swagger UI - Current Server
+    |--------------------------------------------------------------------------
+    |
+    | This allows you to use all servers url in your openapi.json file (Not only local server)
+    | Set this option to false, to use list of servers from openapi.json file
+    |
+    */
+    'local_server_only' => env('SWAGGER_LOCAL_SERVER_ONLY', true),
```

Comment:
can you rename this config variable to `'use_app_url'` / `SWAGGER_UI_USE_APP_URL`

Reply:
ok, did it